get_dependencies.py

```diff
@@ -22,32 +32,42 @@ def download(url, filename):
                 sys.stdout.write('\r[{}{}]'.format('█' * done, '.' * (50 - done)))
                 sys.stdout.flush()
     sys.stdout.write('\n')
+    return filename
+
+
+def sanitize(filename):
+    default_filename = 'file'
```

Comment:
As I said before, we don't want a default filename. If there's no filename found, we want to throw an exception (`raise Exception('No filename could be found for this URL')`.